docs/integrations/ai-engines/monkeylearn.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: MindsDB and MonkeyLearn
+sidebarTitle: MonkeyLearn
+---
+
+MonkeyLearn is a No-code text analysis tool. MindsDB allows you to use pre-built & custom MonkeyLearn models to use its features like classifying text according to user needs and fields of interest like business, reviews, comments, and customer feedback.
+
+## How to bring MonkeyLearn Models to MindsDB
+
+Before creating a model, you will need to create the ML_ENGINE for MonkeyLearn using the `CREATE ML_ENGINE` syntax
+
+```sql
+Create ML_ENGINE monkeylearn
+FROM monkeylearn
+```
+
+<p align="center">
+  <img src="/assets/tutorials/1.create_ml.png" />
+</p>
+
+
+Once the ML_ENGINE is created, we use the `CREATE MODEL` statement to bring MonkeyLearn models to MindsDB.
+
+For this example, you will make use of MonkeyLearn's pre-made model `E-commerce Support Ticket Classifier`.
+
+```sql
+CREATE MODEL mindsdb.ecommerce_ticket_classifier
+PREDICT tag
+USING
+engine = 'monkeylearn',
+api_key = 'your_monkeylearn_api_key',
+model_id = 'model_id',
+input_column = 'text';
+```
+
+On execution, you get:
+
+<p align="center">
+  <img src="/assets/tutorials/createmodel1.png" />
+</p>
+
+Where:
+
+| Expression                             | Description                                              |
+|----------------------------------------|----------------------------------------------------------|
+| `mindsdb.ecommerce_ticket_classifier`  | The model name provided to the model created in MindsDB. |
```

Comment:
Actually, model name is `ecommerce_ticket_classifier` and `mindsdb` is the project name (here the default project). We can mention that.